src/tests/ftest/osa/osa_offline_drain.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+g#!/usr/bin/python
```

Comment:
(pylint-missing-module-docstring) Missing module docstring
(pylint-pointless-statement) Statement seems to have no effect
(pylint-undefined-variable) Undefined variable 'g'